app/assets/v2/js/vue-components.js

```diff
@@ -220,16 +241,15 @@ Vue.component('project-directory', {
 
       let apiUrlProjects = `/api/v0.1/projects_fetch/?${searchParams.toString()}`;
```

Comment:
This endpoint was removed in the URL

Reply:
it is now established through the viewset